node/src/bin/manager.rs

```diff
@@ -966,14 +969,32 @@ async fn main() -> anyhow::Result<()> {
             pending,
             status,
             used,
+            all,
         } => {
             let (primary, store) = if status {
                 let (store, primary) = ctx.store_and_primary();
                 (primary, Some(store))
             } else {
                 (ctx.primary_pool(), None)
             };
-            commands::info::run(primary, store, deployment, current, pending, used)
+
+            Ok(match deployment {
+                Some(deployment) => {
+                    commands::info::run(primary, store, deployment, current, pending, used).err();
+                }
+                None => {
+                    if all {
+                        let deployment = DeploymentSearch::All;
+                        commands::info::run(primary, store, deployment, current, pending, used)
+                            .err();
+                    } else {
+                        eprintln!(
+                            "Please specify a deployment hash or use --all to list all deployments"
+                        );
+                        std::process::exit(1);
```

Comment:
You can just `bail!("Please specify a deployment or use --all to list all deployments)` here (using `anyhow::bail!`) - that's what all other errors end up doing and there's no need to explicitly `exit(1)`